lib/tasks/tools.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :tools do
+  desc "Updates all production builds with 'production' tag"
+  task tag_production_images: :environment do
+    deploys = Stage.select(&:production?).map { |stage| stage.deploys.select(&:succeeded?).first }
+    deploys.each { |deploy| SamsonGcloud::ImageTagger.tag(deploy, stdout: true) }
```

Comment:
- only for stages that don't already have the 'production' permalink
 - not for deploys after the code went live
 - not for deploys >1 year ago ?